.github/workflows/continous-integration-os.yml

```diff
@@ -264,17 +265,6 @@ jobs:
           command: check
           args: --locked --all-features --all-targets
 
-      - name: cargo fetch
-        uses: actions-rs/cargo@v1.0.3
-        with:
-          command: fetch
-
-      - name: Build
-        uses: actions-rs/cargo@v1.0.3
-        with:
-          command: build
-          args: --verbose --release
```

Comment:
I can't talk on Teor's behalf, but when I saw this being removed, and checking the previous steps, it thought this was done as this is not doing any integration test nor any different than what's being validated in the Dockerfile, but to be consistent then the name should also be changed:

https://github.com/ZcashFoundation/zebra/blob/f2a3919d2df0c1db857178f3d41a4f199280dc37/.github/workflows/continous-integration-os.yml#L244

Reply:
We already do similar builds in:

OS:
- Install zebrad from lockfile without cache on ubuntu-latest
- Run tests (because we build the tests)

Docker:
- Both Docker builds (zcash-params and docker/Dockerfile)

So there's no need for a 5th overlapping build step.

Reply:
OK cool, looks good